config/webpack/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const { env, webpackConfig } = require('shakapacker')
+const { existsSync } = require('fs')
+const { resolve } = require('path')
+
+const envSpecificConfig = () => {
+  const path = resolve(__dirname, `${env.nodeEnv}.js`)
+  if (existsSync(path)) {
+    console.log(`Loading ENV specific webpack configuration file ${path}`)
+    return require(path)
+  } else {
+    return webpackConfig
+  }
+}
+
+module.exports = envSpecificConfig()
```

Comment:
We now have a single configuration file.

Here's the shim for loading the older style of a file per value of NODE_ENV, the:

* `/config/webpack/test.js`
* `/config/webpack/development.js`
* `/config/webpack/production.js`